### ISSUE TITLE
Change the Maven identifiers

### ISSUE THREAD

**Issue by abourdon** [cerberustesting/cerberus-source#386]
Two things:

First one:
As Cerberus is an open source project it could be nice to have a more 'open source' groupId, by setting its legacy one `com.redcats`.
I propose `org.cerberus`.

Second one:
Currently, artifactIds contains uppercases which is not conventional. Maven artifactIds should be written by only using lowercases (see http://maven.apache.org/guides/mini/guide-naming-conventions.html for more details).

To sum up, it could be nice to redefine Maven identifiers for our two projects by:
- `org.cerberus:cerberus-testing`
- `org.cerberus:cerberus`
